index.js

```diff
@@ -1,16 +1,13 @@
 'use strict';
 
 module.exports = function getPathOption(options) {
-  let outputPath = 'components';
   if (options === null || typeof options !== 'object') {
     throw new TypeError('getPathOptions first argument must be an object');
   }
-  if (options.path) {
-    outputPath = options.path;
-  } else {
-    if (options.path === '') {
-      outputPath = '';
-    }
+
+  if (options.path || options.path === '') {
```

Comment:
Hmm, what are we actually trying to do here? It seems like we are basically just saying "if I was given `options.path` use it", if thats the case I'd prefer:

```js
if ('path' in options) {
  return options.path;
}
```

Reply:
> Hmm, what are we actually trying to do here?

to be honest, I have no clue. at this point, I'm wondering why we even have a dedicated package for this... I tried to keep the implementation details the exact same for now until we figure out what the desired behavior is.